<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>26</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>26</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="leftRoiBoundLabel">
       <property name="text">
        <string>Left:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="leftRoiBound"/>
     </item>
     <item>
      <widget class="QLabel" name="topRoiBoundLabel">
       <property name="text">
        <string>Top:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="topRoiBound"/>
     </item>
     <item>
      <widget class="QLabel" name="rightRoiBoundLabel">
       <property name="text">
        <string>Right:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rightRoiBound"/>
     </item>
     <item>
      <widget class="QLabel" name="bottomRoiBoundLabel">
       <property name="text">
        <string>Bottom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bottomRoiBound"/>
     </item>
     <item>
      <widget class="QPushButton" name="clearRoiButton">
       <property name="text">
        <string>Clear ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
